csrankings-x.csv

```diff
@@ -245,6 +245,7 @@ Xiaowen Zhang,CUNY,http://www.csi.cuny.edu/faculty/ZHANG_XIAOWEN.html,vX1p5wIAAA
 Xiaoxi He,University of Macau,https://www.fst.um.edu.mo/people/xiaoxi/,jkhl0fUAAAAJ
 Xiaoxing Ma,Nanjing University,http://ics.nju.edu.cn/people/xiaoxingma,44WpWR4AAAAJ
 Xiaoxuan Lu 0001,University of Edinburgh,https://christopherlu.github.io,idu78-EAAAAJ
+Xiaoxue Ren,Zhejiang University,https://scholar.google.com.sg/citations?hl=en&user=mdoN_woAAAAJ,mdoN_woAAAAJ

```

Comment:
Google Scholar page is not suitable as a home page.

Reply:
I have changed her home page in the new commit